graph/pkg/service/v0/service.go

```diff
@@ -50,6 +50,11 @@ func NewService(opts ...Option) Service {
 					r.Get("/", svc.GetGroup)
 				})
 			})
+			r.Route("/drives", func(r chi.Router) {
+				// This route is non-compliant with MS Graph implementation; creating a drive is not supported. There
+				// is no official MS SDK support for this method.
```

Comment:
true but odata specifies how to create new resources in a collection: by sending a POST request with a single element to thet collection endpoint: https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_CreateanEntity

Reply:
> Maybe that API actually exists but is not publicly documented.

This is the direction we're going I think.

Reply:
Funny: already part of the ms spec https://github.com/microsoftgraph/msgraph-metadata/blob/master/openapi/v1.0/openapi.yaml